engine/src/test/scala/cromwell/webservice/EngineStatsActorTest.scala

```diff
@@ -0,0 +1,60 @@
+package cromwell.webservice
+
+import akka.actor.{Actor, ActorRef, Props}
+import akka.testkit.{TestActorRef, TestProbe}
+import cromwell.core.TestKitSuite
+import cromwell.webservice.EngineStatsActor.{EngineStats, JobCount, JobCountQuery}
+import cromwell.webservice.EngineStatsActorTest.{BlockingFakeWorkflowActor, FakeWorkflowActor}
+import org.scalatest.{FlatSpecLike, Matchers}
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+class EngineStatsActorTest extends TestKitSuite("EngineStatsActor") with FlatSpecLike with Matchers {
+  behavior of "EngineStatsActor"
+
+  val replyTo = TestProbe()
+  val defaultTimeout = 100 millis
+
+  it should "return snakeyes with no WorkflowActors" in {
+    TestActorRef(EngineStatsActor.props(List.empty[ActorRef], replyTo.ref))
+    replyTo.expectMsg(defaultTimeout, EngineStats(0, 0))
+  }
+
+  it should "return a single workflow with one job when that's the world it lives in" in {
```

Comment:
In either case, I'm disappointed we didn't continue with https://en.wikipedia.org/wiki/List_of_British_bingo_nicknames